browser.c

```diff
@@ -1011,7 +1021,10 @@ tabbed_launch(void)
     GIOChannel *tabbed_stdout_channel;
     GError *err = NULL;
     gchar *output = NULL;
-    char *argv[] = { "tabbed", "-c", "-d", "-p", "s1", "-n", __NAME__, NULL };
+    char geom[8];
```

Comment:
I don't actually know if `8` is a mart value in this, and/or if there is a better way of converting an int to a string.